ansible_wisdom/ai/api/formatter.py

```diff
@@ -353,15 +353,31 @@ def get_task_names_from_tasks(tasks):
     return names
 
 
-def restore_original_task_names(output_yaml, prompt):
+def restore_original_task_names(output_yaml, prompt, payload_context=''):
     if output_yaml and is_multi_task_prompt(prompt):
+        full_yaml = payload_context + output_yaml
```

Comment:
I've checked with your content by writing it by myself and faced no issue. probably because of the issue that Robin mentioned here: https://github.com/ansible/ansible-ai-connect-service/pull/1013#discussion_r1611491043

I fixed it by returning the same output yaml in case of any YAML load issue not to block the process.